baybe/_optional/onnx.py

```diff
@@ -1,9 +1,26 @@
 """Optional ONNX import."""
 
+from packaging.version import Version
+
 from baybe.exceptions import OptionalImportError
 
 try:
+    import onnx
     import onnxruntime
+
+    # NOTE: In version 1.18.0, the function `onnx.helper.split_complex_to_pairs` was
+    # replaced by the "private" function `_split_complex_to_pairs`.
+    # Not all dependencies, in particulat "skl2onnx" have been updated to use the
+    # new function. To ensure backwards compatibility, we check the version of onnx
+    # and if it is lower than 1.18.0, we import the old function.
```

Comment:
```suggestion
    # new function. 
```

fully obsolete as the lines of code below tell me the very same

Reply:
Removed